builtInFunctions/migrateDataTrie.go

```diff
@@ -77,6 +87,33 @@ func (mdt *migrateDataTrie) ProcessBuiltinFunction(
 	return vmOutput, nil
 }
 
+func (mdt *migrateDataTrie) getAccountForMigration(acntDst vmcommon.UserAccountHandler) (vmcommon.UserAccountHandler, error) {
+	if vmcommon.IsSystemAccountAddress(acntDst.AddressBytes()) {
+		systemSCAccount, err := mdt.loadAccount(vmcommon.SystemAccountAddress)
+		if err != nil {
+			return nil, err
+		}
+
+		return systemSCAccount, nil
+	}
+
+	return acntDst, nil
+}
+
+func (mdt *migrateDataTrie) loadAccount(address []byte) (vmcommon.UserAccountHandler, error) {
+	account, err := mdt.accounts.LoadAccount(address)
```

Comment:
Load accounts return an empty account if the account is not found. Is this ok?

Reply:
I think we could use getExistingAccount here. no need to migrate for non existing accounts.